test/e2e-tests.sh

```diff
@@ -24,7 +24,7 @@
 # project $PROJECT_ID, start Knative eventing system, run the tests and
 # delete the cluster.
 
-source $(dirname $0)/e2e-common.sh
+source "$(dirname "$0")/e2e-common.sh"
```

Comment:
seems like too many `"` here

Reply:
That's not true. It's bash. Bash isn't obvious. That's why everybody should use shellcheck validate their bash code.

Say you have `$0` script as `c:/Users/John Doe/code/eventing/test/e2e-tests.sh`. Running dirname without quotes on that will result in `c:/Users` (`John` will be stripped).